iosApp/iosApp/app/NotificationsApiImpl.swift

```diff
@@ -21,7 +23,52 @@ class NotificationsApiImpl : NSObject, NotificationsApi {
     
     let notificationDelegate = LocalNotificationDelegate()
     
-    func createLocalNotification(title: String, message: String, timeInMS: Int64, notificationId: Int32) {
+    
+    func createReminderNotificationsForSessions(sessions: [MySessions]) {
+        if(NotificationsKt.reminderNotificationsEnabled() && NotificationsKt.notificationsEnabled()) {
+            for session in sessions {
+                let notificationTime = session.startsAt.toLongMillis() - Int64(AppContext().TEN_MINS_MILLIS)
+                if (notificationTime > FunctionsKt.currentTimeMillis()) {
```

Comment:
Does this mean we get a notification for every event which is more than 10 min away? or does this only get called with sessions which are happening soon?

Reply:
Yes you get a notification for every event in the future. This is scheduling a notification to go off ten minutes before each event. So if you've RSVPed to events at 1 and 2, then you get notifications at 12:50 and 1:50.

Reply:
ah, yes, my bad